README.md

```diff
@@ -94,7 +100,7 @@ Note the Arn returned from the command `POLICY_ARN`
       --user-pool-id XXXXXXXX \
       --client-name lambda-migration-client \
       --no-generate-secret \
-      --explicit-auth-flows "ALLOW_USER_PASSWORD_AUTH" "ALLOW_ADMIN_USER_PASSWORD_AUTH""
+      --explicit-auth-flows "ALLOW_USER_PASSWORD_AUTH" "ALLOW_ADMIN_USER_PASSWORD_AUTH" "ALLOW_REFRESH_TOKEN_AUTH"
```

Comment:
Why?

Reply:
Caught me by surprise, it's required in AWS 
```
11:54 $ aws cognito-idp create-user-pool-client \
>       --user-pool-id XXXXXXXXXX \
>       --client-name lambda-migration-client \
>       --no-generate-secret \
>       --explicit-auth-flows "ALLOW_USER_PASSWORD_AUTH" "ALLOW_ADMIN_USER_PASSWORD_AUTH"

An error occurred (InvalidParameterException) when calling the CreateUserPoolClient operation: ALLOW_REFRESH_TOKEN_AUTH should always be an accepted auth flow.
```

Should always be an accepted auth flow appears to mean required